src/AutoRest.CSharp/LowLevel/AutoRest/LowLevelTarget.cs

```diff
@@ -41,6 +44,21 @@ public static void Execute(GeneratedCodeWorkspace project, InputNamespace inputN
             var optionsWriter = new CodeWriter();
             ClientOptionsWriter.WriteClientOptions(optionsWriter, library.ClientOptions);
             project.AddGeneratedFile($"{library.ClientOptions.Type.Name}.cs", optionsWriter.ToString());
+
+            await project.PostProcess(PostProcess);
+        }
+
+        private static async Task<Project> PostProcess(Project project)
+        {
+            if (Configuration.KeepUnusedModels)
+                return project;
+
+            var postProcessor = new DpgPostProcessor();
+            project = await postProcessor.InternalizeAsync(project);
```

Comment:
I assume this two step process here works like the following:

If we have 3 types that get internalized
Foo.cs
Bar.cs
BangListResult.cs

step one will mark all 3 as internal

step two will delete Foo and Bar, but since BangListResult has internal references it will be left behind?

Reply:
Yes. This is how it works.
Now that since I have combined maybe I could let the remover to go first and by default the internalizer will have less classes to look at which might slightly accelerate the process. What do you think?